include/igl/decimate.cpp

```diff
@@ -185,7 +185,13 @@ IGL_INLINE bool igl::decimate(
       cost_and_placement(e,V,F,E,EMAP,EF,EI,cost,p);
       C.row(e) = p;
       costs(e) = cost;
-    },10000);
+    },
+#ifndef NDEBUG
+    SIZE_MAX
```

Comment:
This should be superseded by the new way of setting the number of parallel for threads. I wonder if that method should default to serial in debug mode? (obviously also settable via the env variable, but might be nice to set defaults to the common case if we indeed think debugging in serial is what's generally desired).

Reply:
I don't think debugging in serial should be the default, as it would be tricky to debug race conditions that occur due to parallel for loops. But yes if the goal of this ifdef was to run parallel code in serial, then the new env var method should be preferred to set the number of threads.